apps/web/src/components/nav/EnvironmentSelect/EnvironmentSelect.tsx

```diff
@@ -9,6 +9,7 @@ export const EnvironmentSelectRenderer: React.FC<ReturnType<typeof useEnvironmen
   isPopoverOpened,
   setIsPopoverOpened,
   handlePopoverLinkClick,
+  readonly,
```

Comment:
`readonly` has been passed into the Select props, which do not have such prop - this removes the console error and also removes the concern that if select will have that property added, it wouldn't make the env selects disabled when its value is production